apps/laboratory/src/components/Ethers/EthersSendCallsWithPaymasterServiceTest.tsx

```diff
@@ -12,6 +12,8 @@ import {
   getCapabilitySupportedChainInfo
 } from '../../utils/EIP5792Utils'
 import { W3mFrameProvider } from '@reown/appkit-wallet'
+import { abi, address as donutAddress } from '../../utils/DonutContract'
+import { ethers } from 'ethers5'
```

Comment:
You're updating the `ethers` examples but importing from `ethers5`, could you align the examples with the right package? I think it'd still work but we'd better keep them as expected.

Also, could you add same example for `ethers5` separately? -> `Ethers5SendCallsWithPaymasterServiceTest` (ethers5 in the naming)